tests/pipelines/test_pipelines_common.py

```diff
@@ -2153,6 +2157,41 @@ def test_StableDiffusionMixin_component(self):
             )
         )
 
+    @require_torch_gpu
+    @slow
+    def test_torch_compile_recompilation(self):
+        inputs = self.get_dummy_inputs(torch_device)
+        components = self.get_dummy_components()
+
+        pipe = self.pipeline_class(**components).to(torch_device)
+        if getattr(pipe, "unet", None) is not None:
+            pipe.unet = torch.compile(pipe.unet, fullgraph=True)
+        else:
+            pipe.transformer = torch.compile(pipe.transformer, fullgraph=True)
+
+        with torch._dynamo.config.patch(error_on_recompile=True):
+            _ = pipe(**inputs)
+
+    @require_torch_gpu
+    @slow
+    def test_torch_compile_graph_breaks(self):
+        # Inspired by:
+        # https://github.com/pytorch/pytorch/blob/916e8979d3e0d651a9091732ce3e59da32e72b0e/test/dynamo/test_higher_order_ops.py#L138
+        counters.clear()
+
+        inputs = self.get_dummy_inputs(torch_device)
+        components = self.get_dummy_components()
+
+        pipe = self.pipeline_class(**components).to(torch_device)
+        if getattr(pipe, "unet", None) is not None:
+            pipe.unet = torch.compile(pipe.unet, fullgraph=True)
+        else:
+            pipe.transformer = torch.compile(pipe.transformer, fullgraph=True)
+
+        _ = pipe(**inputs)
+        num_graph_breaks = len(counters["graph_break"].keys())
+        self.assertEqual(num_graph_breaks, 0)
```

Comment:
`fullgraph=True` is enough to ensure that test fails if there are graph breaks. So, you can remove this check.